model_bakery/utils.py

```diff
@@ -17,6 +19,12 @@ def import_from_str(import_string: Optional[Union[Callable, str]]) -> Any:
     """
     if isinstance(import_string, str):
         path, field_name = import_string.rsplit(".", 1)
+
+        # is it a Django model in the app_name.ModelName convention?
```

Comment:
Yep.

We could import Django here. This way the import will happen during execution time, not interpretation. Hacky, I know, but that's all we can do so far. The long-term fix is to implement a better way to import the `seq` function as `model_bakery.seq` rather than importing it directly from the `utils` file.